hedera-node/src/main/java/com/hedera/services/sigs/order/HederaSigningOrder.java

```diff
@@ -944,10 +944,21 @@ private boolean hasNondeprecatedAdminKey(ContractUpdateTransactionBody op) {
 			TransactionID txnId,
 			AccountID target,
 			SigningOrderResultFactory<T> factory
+	) {
+		return forSingleAccount(txnId, target, factory, false);
+	}
+
+	private <T> SigningOrderResult<T> forSingleAccount(
+			TransactionID txnId,
+			AccountID target,
+			SigningOrderResultFactory<T> factory,
+			boolean usePure
```

Comment:
As above, this pure/impure distinction is no longer necessary when we accept the 0.15.0 implementation of `BackingAccounts`.